admin.go

```diff
@@ -0,0 +1,376 @@
+package sarama
+
+import "errors"
+
+type ClusterAdmin interface {
```

Comment:
Sorry my previous comment wasn't clear. I still think we should have a description for `ClusterAdmin`, I just think it doesn't need to mention the bit about "This client was introduced in 0.11.0.0 and the API is still evolving."